gtwrap/pybind_wrapper.py

```diff
@@ -51,7 +51,8 @@ def _py_args_names(self, args_list):
                     arg.default = "\"{arg.default}\"".format(arg=arg)
                 argument = 'py::arg("{name}"){default}'.format(
                     name=arg.name,
-                    default=' = {0}'.format(arg.default) if arg.default else '')
+                    default=' = {0}'.format(arg.default)
+                    if arg.default else '')
```

Comment:
autoformat

Reply:
I don't understand this

Reply:
See other reply and feel free to ignore - I thought this was an unintentional change that got lumped in with a commit.